test/e2e/specs/site-editor/style-book.spec.js

```diff
@@ -59,37 +59,50 @@ test.describe( 'Style Book', () => {
 		).toBeVisible();
 		await expect( page.locator( 'role=tab[name="Theme"i]' ) ).toBeVisible();
 
+		// Buttons to select block examples are rendered within the Style Book iframe.
+		let styleBookIframeLocator = page.frameLocator(
+			'.edit-site-style-book__iframe'
```

Comment:
Is there a `name` attribute for the frame? Maybe it should? If so we can just do `page.frameLocator( '[name="name"]' )` or `page.frame( name )` here.

Reply:
The `page.frameLocator( '[name="name"]' )` approach worked nicely here, I've added a `style-book-canvas` name to the iframe. When testing locally, I noticed that `page.frameLocator()` appears to support reevaluating the iframe as needed, however `page.frame()` appeared to grab a specific instance of a frame, so failed after we switch tabs. I've gone with `page.frameLocator` in both tests for consistency, but targeted by name.